portfolio/src/main/java/com/google/sps/data/ScheduleRequest.java

```diff
@@ -0,0 +1,69 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.Collection;
+/**
```

Comment:
You might need to import CalendarEvent and Task?

Reply:
Not necessary because of the package.